python/cudf/cudf/tests/test_series.py

```diff
@@ -2848,3 +2848,13 @@ def test_nans_to_nulls_noop_copies_column(value):
     ser1 = cudf.Series([value])
     ser2 = ser1.nans_to_nulls()
     assert ser1._column is not ser2._column
+
+
+@pytest.mark.parametrize("dropna", [False, True])
+def test_nunique_all_null(dropna):
+    data = [np.nan, np.nan]
+    pd_ser = pd.Series(data)
+    cudf_ser = cudf.Series(data, nan_as_null=True)
```

Comment:
I don't think we are actually testing `nan` case here or do you want to test only `NA` cases?

Reply:
I was hoping to just test the NA case here i.e. where `dropna` would impact the nunique count

Reply:
ah, okay. Can we use `None` then instead of `np.nan`?

Reply:
Sure thing. Changed to use None